src/content/concepts/module-federation.md

```diff
@@ -120,3 +125,101 @@ Each page of a Single Page Application is exposed from container build in a sepa
 ### Components library as container
 
 Many applications share a common components library which could be built as a container with each component exposed. Each application consumes components from the components library container. Changes to the components library can be separately deployed without the need to re-deploy all applications. The application automatically uses the up-to-date version of the components library.
+
+## Troubleshooting
+
+`Uncaught Error: Shared module is not available for eager consumption` <br/> The application is eagerly executing an app which is operating as an omnidirectional host. There are options to choose from: <br> You can set the dependency as eager inside the advanced API of Module Federation, which doesn’t put the modules in an async chunk, but provides them synchronously.<br>This allows us to use these shared modules in the initial chunk. But be careful as all provided and fallback modules will always be downloaded. <br> It’s wise to provide it only at one point of your app, e.g. the shell.
+
+
+We strongly recommend using an async boundary. Doing so will not create any additional round trips, it’s also more performant in general as initialization code is split out of a larger chunk.
+`ìmport('./bootstrap')`
+
+Create a `bootstrap.js` file, and move the original contents of the entry point code into that file.
+
+```js
+//index.js
+import('./bootstrap');
+//bootstrap.js
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+ReactDOM.render(<App />, document.getElementById('root'));
+```
+
+Methods mentioned above work, but can have some limits or drawbacks.
+
+```js
+new ModuleFederationPlugin({
+  shared: {
+    ...deps,
+    react: {
+      eager: true,
+    }
+  }
+});
+```
+
+`bundle-loader` can be used as an alternative to setting dependencies as "eager". This method is less performant as it will introduce additional round trips.
```

Comment:
```suggestion
`bundle-loader` can be used as an alternative to setting dependencies as `'eager'`. This method is less performant as it will introduce additional round trips.
```